src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -99,6 +93,19 @@ def get_registry_by_name(cli_ctx, registry_name, resource_group_name=None):
     return client.get(resource_group_name, registry_name), resource_group_name
 
 
+def get_resource_id_by_login_server(client, login_server):
+    """Returns the resource ID for the container registry.
+    :param str login_server: The login server of the container registry.
+    """
+    from .custom import acr_list
+    registry_list = acr_list(client)
+    elements = [item for item in registry_list if item.login_server.lower() == login_server.lower()]
```

Comment:
Create a var = login_server.lower() and reuse it, rather than calling lower() every time.